app/src/components/TaskBoard.js

```diff
@@ -24,7 +25,12 @@ function TaskBoard({
   }, []);
 
   return (
-    <div className="taskBoard">
+    <div
+      className="taskBoard"
+      style={{
+        display: tab === "taskboard" ? "block" : "none",
+      }}
```

Comment:
Suggestion: similar to my comment above about making the className dynamic instead of the styles, you could have a `hidden` className which is added if the taskboard is not the currently active thing.  The default display for the component is `block`, but is `none` for the `hidden` class.  This is a commonly-used pattern in production code.